Gemfile.lock

```diff
@@ -1,78 +1,109 @@
 GEM
```

Comment:
Remove Gemfile.lock from repository and add .lock files .gitignore.

Reply:
done, please check